hotham/src/contexts/physics_context.rs

```diff
@@ -1,4 +1,3 @@
-use crossbeam::channel::Receiver;
```

Comment:
This can be replaced with `rapier3d::crossbeam` if we don't want to depend on `crossbeam` directly.